dayforce-to-msad/main.bal

```diff
@@ -156,9 +155,9 @@ function transform(dayforce:Employee employee) returns ADEmployeeUpdate|error =>
         middleName: employee.MiddleName,
         sn: employee.LastName,
         displayName: employee.DisplayName,
-        mobile: getContactInfo(personContactItems, BUSINESS_MOBILE),
-        telephoneNumber: getContactInfo(personContactItems, BUSINESS_PHONE),
-        mail: getContactInfo(personContactItems, BUSINESS_EMAIL),
+        mobile: employee?.HomeOrganization?.ContactCellPhone,
+        telephoneNumber: employee?.HomeOrganization?.BusinessPhone,
+        mail: employee?.HomeOrganization?.ContactEmail,
```

Comment:
Shall we extract `employee?.HomeOrganization` out to a variable?

Reply:
Extracted in [358fa6c](https://github.com/MaryamZi/ballerina-sync-samples/pull/1/commits/358fa6cf567f0c398fddd86749ff2898d45178c4)